src/Raven.Studio/wwwroot/App/views/database/settings/customAnalyzers.html

```diff
@@ -1,12 +1,16 @@
 <div class="flex-vertical absolute-fill content-margin custom-analyzers">
     <div class="row flex-row flex-grow flex-stretch-items">
         <div class="col-sm-12 col-lg-6 flex-vertical">
-            <div class="flex-header">
+            <div class="flex-header flex-horizontal">
                 <div>
                     <a class="btn btn-primary" data-bind="attr: { href: addUrl }, requiredAccess: 'ReadWrite'">
                         <i class="icon-plus"></i><span>Add a Database Custom Analyzer</span>
                     </a>
                 </div>
+                <div class="flex-separator"></div>
+                <small title="Go to the RavenDB analyzers documentation">
+                    <a target="_blank" data-bind="attr: { href: 'https://ravendb.net/l/VWCQPI' }"><i class="icon-link"></i>Analyzers tutorial</a>
```

Comment:
client version is missing.

Reply:
see: https://github.com/ravendb/ravendb/blob/7bd7086fd527a3accab0820ef23ec7f41d76158d/src/Raven.Studio/typescript/viewmodels/database/query/query.ts#L625

Reply:
done